pants.toml

```diff
@@ -224,7 +223,7 @@ args = [
 ]
 
 [pytest]
-install_from_resolve = "pytest"
+install_from_resolve = "st2"
```

Comment:
This is the line that tells pants to use `lockfiles/st2.lock` instead of `lockfiles/pytest.lock` (which is deleted above).